kmmo/README.md

```diff
@@ -42,6 +42,8 @@ $ oc label node <node_name> intel.feature.node.kubernetes.io/dgpu-canary=true
 ```
 
 3.	Use pre-build mode to deploy the driver container.
+
+Note: Due to a known KMM regression issue, please follow the steps in [Workaround section](https://github.com/intel/intel-technology-enabling-for-openshift/issues/194) before running below command.
```

Comment:
Note: If the KMM Operator version < 2.0.2, please use the steps in the workaround section of https://github.com/intel/intel-technology-enabling-for-openshift/issues/194  to workaround this known issue

Reply:
Done